uwblueprint/childrens-aid-society
# Review Case - Court Information

## Notion ticket link
[Review Case - Court Information](https://www.notion.so/uwblueprintexecs/Review-Case-Court-Information-94c0781aa66f4520a2da35df539168e2)



## Implementation description
* Implemented UI for Court Information on the Case Review page
* Modified other components like Autocomplete and CustomSelectField to include optional readOnly parameter
<img width="1129" alt="Screen Shot 2023-02-06 at 11 06 58 AM" src="https://user-images.githubusercontent.com/66758256/217023149-772b62f7-f53e-40de-ae03-33dd9395b4c0.png">



## Steps to test
1. Go to localhost:3000/intake, fill out the form (especially the Court Information section), go to review
2. Check if design is correct
3. Make sure user can modify and interact with the fields on the Court Information page
4. Make sure user cannot modify or interact with the fields on the Case Review page



## What should reviewers focus on?
* Design
* Check the modifications for Autocomplete.tsx and CustomSelectField.tsx
* Not sure if the conditional cases (eg. 'Attach document' vs 'Download attachment' implementation in CourtInformationForm.tsx) could be handled more elegantly, thoughts on this syntax?



## Checklist
